lib/rules/web/admin_console/4.3/logs.xyaml

```diff
@@ -0,0 +1,11 @@
+# logs page
+switch_to_second_container:
+  elements:
+  - selector:
+      xpath: //button[contains(@class,'pf-c-dropdown__toggle')]//span[contains(text(),'<current_container_name>')]
+    op: click
+  action: check_page_contains
```

Comment:
You didn't pass parameter in scenario for check_page_contains, and also this action is not a must.

Reply:
will remove